alembic/versions/b74cc308b1ec_move_responses_to_db.py

```diff
@@ -0,0 +1,83 @@
+revision = 'b74cc308b1ec'
+down_revision = '66d78d1a29ee'
+branch_labels = None
+depends_on = None
+
+import alembic
+import sqlalchemy
+import json
+from collections import defaultdict
+from www.commands import build_response_dict
+
+def upgrade():
+	commands = alembic.op.create_table("commands",
+		sqlalchemy.Column("id", sqlalchemy.Integer, primary_key=True, autoincrement=True),
+		sqlalchemy.Column("access", sqlalchemy.Integer, nullable=False),
+	)
+
+	aliases = alembic.op.create_table("commands_aliases",
+		sqlalchemy.Column("id", sqlalchemy.Integer, primary_key=True, autoincrement=True),
+		sqlalchemy.Column("command_id", sqlalchemy.Integer,
+			sqlalchemy.ForeignKey("commands.id", ondelete="CASCADE"), nullable=False),
+		sqlalchemy.Column("alias", sqlalchemy.String(255), nullable=False),
+	)
+	alembic.op.create_index("commands_aliases_alias", "commands_aliases", ["alias"], unique=True)
+
+	responses = alembic.op.create_table("commands_responses",
+		sqlalchemy.Column("id", sqlalchemy.Integer, primary_key=True, autoincrement=True),
+		sqlalchemy.Column("command_id", sqlalchemy.Integer,
+			sqlalchemy.ForeignKey("commands.id", ondelete="CASCADE"), nullable=False),
+		sqlalchemy.Column("response", sqlalchemy.String(512), nullable=False),
+	)
+
+	datafile = alembic.context.config.get_section_option("lrrbot", "datafile", "data.json")
+	with open(datafile) as f:
+		data = json.load(f)
+
+	command_data = defaultdict(list)
+	for cmd, val in data["responses"].items():
+		resp = tuple(val["response"]) if isinstance(val["response"], list) else (val["response"],)
+		access = {"any": 0, "sub": 1, "mod": 2}[val["access"]]
+		command_data[resp, access].append(cmd)
+
+	command_records = [{}]
+	alias_records = []
+	response_records = []
+
+	for (resp, access), cmds in command_data.items():
+		if "advice" in cmds:
+			command_records[0] = {"access": access}
+			command_id = 1
```

Comment:
I don't like hardcoding `!advice` as command ID 1 because it's not guaranteed that `!advice` exists (for example when following the setup instructions) and neither that command ID 1 is actually `!advice` (for example because we've swapped `!advice` and `!advice2`).